bar.go

```diff
@@ -118,8 +118,13 @@ func (bar *Bar) play(cur int64) (n int, err error) {
 	last := bar.percent
 	bar.step = cur
 	bar.percent = bar.getPercent()
-
-	if bar.percent > last {
+	
+	if cur == bar.max {
+		remain := (150 - len(bar.rate)) / 3
```

Comment:
should it no be `bar.step` instead of 150 ? what does 150 stands for ?

Reply:
150 stand for bar.rate characters total length when progress is 100% , because one character "█".length == 3,
In order to calculate how many more "█" are needed to fill the progress